packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -0,0 +1,140 @@
+import { ICheckboxStyles } from './Checkbox.Props';
+import {
+  ITheme,
+  mergeStyleSets,
+  getFocusStyleBasedOnAncestorClass
+} from '../../Styling';
+import { memoizeFunction } from '../../Utilities';
+
+const MS_CHECKBOX_LABEL_SIZE = '20px';
+const MS_CHECKBOX_TRANSITION_DURATION = '200ms';
+const MS_CHECKBOX_TRANSITION_TIMING = 'cubic-bezier(.4, 0, .23, 1)';
+const MS_CHECKBOX_HIGHCONTRAST_ICONBOXSIZE = '16px';
+
+const MS_HIGHCONTRAST_ACTIVE = '@media screen and (-ms-high-contrast: active)';
+const MS_HIGHCONTRAST_BLACK_ON_WHITE = '@media screen and (-ms-high-contrast: black-on-white)';
+
+export const getStyles = memoizeFunction((
+  theme: ITheme,
+  customStyles?: ICheckboxStyles
+): ICheckboxStyles => {
+  const { semanticColors, palette } = theme;
+  const checkmarkFontColor = semanticColors.bodyBackground;
+  const checkboxBorderColor = semanticColors.inputBorder;
+  const checkboxHoveredBorderColor = semanticColors.inputBorderHovered;
+  const checkboxCheckedBackground = semanticColors.inputBackgroundChecked;
+  const checkboxCheckedHoveredBackground = semanticColors.inputBackgroundCheckedHovered;
+  const checkboxDisabledBackground = semanticColors.disabledText;
+  const checkboxDisabledTextColor = semanticColors.disabledText;
+
+  const styles: ICheckboxStyles = {
+    root: {
+      overflow: 'hidden',
+      position: 'relative',
+    },
+    label: [
+      getFocusStyleBasedOnAncestorClass(theme, '.ms-Fabric.is-focusVisible .is-inFocus'),
+      {
+        display: 'inline-flex',

```

Comment:
Yes, this is to avoid the focus border from spanning the entire checkbox component. Instead with this it will just span the  whole text and the checkbox. Please see below.

With display:'inline-flex'
![image](https://user-images.githubusercontent.com/2162858/28284780-19630430-6ae7-11e7-8cb6-f117937d14a9.png)

With display:'flex'
![image](https://user-images.githubusercontent.com/2162858/28284824-4473d9ce-6ae7-11e7-962a-51525cbd710b.png)